src/pages/Staff/Scripts/ScriptModerationScreen.tsx

```
@@ -65,31 +73,37 @@ Tóm tắt và lời kêu gọi hành động...`,
+  );
+  const [searchQuery, setSearchQuery] = useState("");
+  const [selectedScript, setSelectedScript] = useState<PendingScript | null>(
+    null,
+  );
   const [actionLoading, setActionLoading] = useState<string | null>(null);
 
-  const filtered = scripts.filter(s => {
-    const matchFilter = filter === 'pending' || s.id === s.id; // simple filter
-    const matchSearch = !searchQuery.trim() || s.title.toLowerCase().includes(searchQuery.toLowerCase());
+  const filtered = scripts.filter((s) => {
+    const matchFilter = filter === "pending" || s.id === s.id; // simple filter
```

Biome (2.4.11)

[error] Comparing to itself is potentially pointless.

(lint/suspicious/noSelfCompare)